spmd/tensor/parallel/_view_with_dim_change.py

```diff
@@ -37,6 +37,7 @@ def forward(  # type: ignore
         ctx.previous_placement = self.placements
         ctx.previous_device_mesh = self.device_mesh
         ctx.previous_local_shape = self.to_local().size()
+        ctx.previous_logical_shape = self.size()
```

Comment:
Is logical shape the same as "global" shape?

Reply:
global makes more sense to me but i'm okay either way, as long as we're consistent.